### PR TITLE
add bitnami repo

### PR DESCRIPTION
**Pull request by woodgear**

### CODE REVIEW

**Review by iflan7744 — APPROVED** [review]

**Comment by bridgetkromhout** [comment]
Hi, @woodgear - thanks for this edit to the German-language translation of the "using helm" page. Can you please also make this change to the English page at https://github.com/helm/helm-www/blob/main/content/en/docs/intro/using_helm.md ? (Other languages are okay too, but we want to make sure the English page gets any essential clarifications, since people creating new translations are likely starting there.) Thanks!

File: content/de/docs/intro/using_helm.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ Am einfachsten hat das Kommando zwei Argumente: Einen Versionsnamen, den Sie gew
 haben und den Namen des Charts, dass Sie installieren möchten.
 
 ```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
```
Comment (bacongobbler) [inline content/de/docs/intro/using_helm.md:100]: The call to `helm repo update` after a `helm repo add` is redundant. Would you mind removing this?

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by bridgetkromhout** [comment]
Hi, @woodgear - we would like to merge your PR - can you please make the requested changes? If you cannot, we make have to close it unmerged. Thanks.

**Comment by bridgetkromhout** [comment]
Hi, @woodgear - we are going to close this PR unmerged for now. Please re-open with the requested changes if desired! Thanks.

**Pull request closed without merging**